arm/Microsoft.Network/connections/readme.md

```diff
@@ -21,27 +21,67 @@ This template deploys a virtual network gateway connection.
 
 **Optional parameters**
 | Parameter Name | Type | Default Value | Allowed Values | Description |
 | :-- | :-- | :-- | :-- | :-- |
 | `customIPSecPolicy` | object | `{object}` |  | The IPSec Policies to be considered by this connection |
 | `enableBgp` | bool | `False` |  | Value to specify if BGP is enabled or not |
 | `enableDefaultTelemetry` | bool | `True` |  | Enable telemetry via the Customer Usage Attribution ID (GUID). |
+| `localNetworkGateway2` | object | `{object}` |  | The local network gateway. Used for connection type [IPsec] |
 | `location` | string | `[resourceGroup().location]` |  | Location for all resources. |
 | `lock` | string | `'NotSpecified'` | `[CanNotDelete, NotSpecified, ReadOnly]` | Specify the type of lock. |
-| `remoteEntityResourceGroup` | string | `''` |  | Remote Virtual Network Gateway/ExpressRoute resource group name |
-| `remoteEntitySubscriptionId` | string | `''` |  | Remote Virtual Network Gateway/ExpressRoute Subscription ID |
-| `routingWeight` | string | `''` |  | The weight added to routes learned from this BGP speaker. |
+| `peer` | object | `{object}` |  | The remote peer. Used for connection type [ExpressRoute] |
+| `routingWeight` | int | `-1` |  | The weight added to routes learned from this BGP speaker. |
 | `tags` | object | `{object}` |  | Tags of the resource. |
 | `usePolicyBasedTrafficSelectors` | bool | `False` |  | Enable policy-based traffic selectors |
-| `virtualNetworkGatewayConnectionType` | string | `'Ipsec'` | `[Ipsec, VNet2VNet, ExpressRoute, VPNClient]` | Gateway connection type. |
+| `virtualNetworkGateway2` | object | `{object}` |  | The remote Virtual Network Gateway. Used for connection type [Vnet2Vnet] |
+| `virtualNetworkGatewayConnectionType` | string | `'IPsec'` | `[IPsec, Vnet2Vnet, ExpressRoute, VPNClient]` | Gateway connection type. |
 | `vpnSharedKey` | string | `''` |  | Specifies a VPN shared key. The same value has to be specified on both Virtual Network Gateways |
 
 
+### Parameter Usage: `virtualNetworkGateway1`
+
+The primary virtual network gateway object.
+
+```json
+"virtualNetworkGateway1" : {
+    "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Network/virtualNetworkGateways/myGateway",
+}
+```
+
+### Parameter Usage: `virtualNetworkGateway2`
+
+The seconday virtual network gateway used for VNET to VNET connections.
+
+```json
+"virtualNetworkGateway1" : {
+    "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Network/virtualNetworkGateways/myGateway",
+}
+```
+
+### Parameter Usage: `localNetworkGateway2`
+
+The local virtual network gateway object.
+
+```json
+"localNetworkGateway2" : {
+    "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Network/localNetworkGateways/myGateway",
+}
+```
+
+### Parameter Usage: `localNetworkGateway2`
+
+The local virtual network gateway object.
```

Comment:
Please elaborate on the difference between this and the previous localNetworkGateway parameter usage